src/main/suitable/compliment/sources/cljs/analysis.clj

```diff
@@ -66,10 +66,11 @@
   "Returns a map {ns-name-or-alias ns-name} for the given namespace."
   [env ns]
   (when-let [found (find-ns env ns)]
-    (let [imports (:imports found)]
+    (let [imports    (:imports found)
+          as-aliases (get found :as-aliases {})]
       (->> (:requires found)
            (filter #(not (contains? imports (key %))))
-           (into {})))))
+           (into as-aliases)))))
```

Comment:
We could use transfucer form at this point

Reply:
(while we're here filter+not -> remove)

Reply:
Good idea. Done!